docker/mariadb/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# syntax=docker/dockerfile:1.4
+FROM mariadb:10.11.4
+
+WORKDIR /docker-entrypoint-initdb.d
```

Comment:
`/docker-entrypoint-initdb.d` に置かれたスクリプトが初回起動時に実行されるというのはイメージの非自明な仕様なので、[image のドキュメント](https://hub.docker.com/_/mariadb#) 内でのこの仕様に関する言及を探せるようにコメントを書いてください。この仕様は、該当ドキュメントの `Initializing a fresh instance` のセクションで言及されています。
   > Furthermore, it will execute files with extensions `.sh`, `.sql`, `.sql.gz`, `.sql.xz` and `.sql.zst` that are found in `/docker-entrypoint-initdb.d`

Reply:
また、固定の `gachadata.sql` ならば構わないのですが、最新の `gachadata.sql` を持ってくるような実装に切り替えた場合、初回起動時にしか初期化スクリプトが実行されない `/docker-entrypoint-initdb.d` に置くのだと困りそうです。

Reply:
> また、固定の `gachadata.sql` ならば構わないのですが、最新の `gachadata.sql` を持ってくるような実装に切り替えた場合、初回起動時にしか初期化スクリプトが実行されない `/docker-entrypoint-initdb.d` に置くのだと困りそうです。

これなんですが、`gachadata.sql`はべき等ではなく毎回上書きするわけにもいかない(例えば、検証用のデータを入れ込み、再起動したい時などに困る)ので、最新の`gachadata.sql`を適用する時にはdocker volumeの削除などを行ってほしいという意図があります。

Reply:
確かに！「初回起動時には必ず最新版の `gachadata.sql` が適用され、以後基本的に上書きはされないが、特定のシェルスクリプトを一回走らせると `gachadata.sql` が変更してた table を一旦 `TRUNCATE` したうえで最新版の `gachadata.sql` が適用される」とかだとどうでしょうか？

Reply:
その方法であれば、2つの問題が両方解決できるので良いと思います！
やってみます。